app-benchmarks/stress-ng/stress-ng-0.09.53.ebuild

```diff
@@ -4,8 +4,8 @@
 EAPI="7"
 
 DESCRIPTION="Stress test for a computer system with various selectable ways"
-HOMEPAGE="https://kernel.ubuntu.com/~cking/stress-ng/"
-SRC_URI="https://kernel.ubuntu.com/~cking/tarballs/${PN}/${P}.tar.xz"
+HOMEPAGE="http://kernel.ubuntu.com/~cking/stress-ng/"
```

Comment:
why?

Reply:
oO Sorry, absolutly overseen this. Corrected both now.